census_data_aggregator/__init__.py

```diff
@@ -347,3 +347,42 @@ def approximate_product(pair_one, pair_two):
 
     # Return the results
     return product_estimate, product_moe
+
+def approximate_percentchange(pair_one, pair_two):
+    """
+    Calculates the percent change from one time period to another and approximates its margin of error.
+
+    Follows the U.S. Census Bureau's `official guidelines`_.
+
+    Args:
+        pair_one (list): a two-item sequence with a U.S. Census
+            bureau estimate and its margin of error.
+
+        pair_two (list): a two-item sequence with a U.S. Census
+            bureau estimate and its margin of error.
+
+    Returns:
+        A two-item sequence containing the estimate and its approximate margin of error.
+
+    Examples:
+        
+
```

Comment:
Let's get an example in here. Maybe we start with one of the other examples from another function and adapt it?